lib/util/middleware.js

```diff
@@ -90,3 +92,53 @@ export async function updateUser (middlewareConfig, token, data) {
     }
   }
 }
+
+export async function getTrips (middlewareConfig, token) {
+  const { apiBaseUrl, apiKey } = middlewareConfig
+  const requestUrl = `${apiBaseUrl}${API_MONITORTRIP_PATH}`
+
+  return secureFetch(requestUrl, token, apiKey, 'GET')
+}
+
+export async function addTrip (middlewareConfig, token, data) {
+  const { apiBaseUrl, apiKey } = middlewareConfig
+  const requestUrl = `${apiBaseUrl}${API_MONITORTRIP_PATH}`
+
+  return secureFetch(requestUrl, token, apiKey, 'POST', {
+    body: JSON.stringify(data)
+  })
+}
+
+export async function updateTrip (middlewareConfig, token, data) {
+  const { apiBaseUrl, apiKey } = middlewareConfig
+  const { id } = data // Middleware ID, NOT auth0 (or similar) id.
```

Comment:
Is this comment about middleware ID necessary? Seems like copy pasta from user stuff.

Reply:
It's copy pasta.